drivers/ieee802154/ieee802154_nrf5.c

```diff
@@ -46,7 +46,11 @@ struct nrf5_802154_config {
 
 static struct nrf5_802154_data nrf5_data;
 
-#define ACK_TIMEOUT K_MSEC(10)
+/* Increase ACK timeout to cover the maximum CSMA CA time in default
```

Comment:
Not sure about this comment. Is it asking owners to increase the value if needed, then please add TODO. If it is explaining why you changed 10 to 40 I would focus more on explaining why 40 not why it was extended. eg. time value includes cca time + transmission time + ack max delay + ack receive time. Potential user will not know of the old value until he looks at the change history.

Reply:
BTW. 40ms seems quite a lot of time is it really that long?

Reply:
It's a temporary solution, there's a TODO below:
```
TODO Switch to ACK timeout at driver level.
```
Why 40ms? I was told that in the default configuration the worst-case scenario for CSMA CA is ~30-something ms.